### PR TITLE
Publish Porter docker image

### PR DESCRIPTION
**Pull request by derekpierre**
**Type of PR:**
- [ ] Bugfix
- [ ] Feature
- [ ] Documentation
- [X] Other

**Required reviews:** 
- [ ] 1
- [X] 2
- [ ] 3

Based over #2732 .

**Issues fixed/closed:**
Related to #2669 , #2724 .

**Why it's needed:**
We need to publish a docker image for porter specifically.

**Notes for reviewers:**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -911,6 +968,32 @@ jobs:
...
+            set +o pipefail
+            docker load -i ~/docker/nucypher-porter.tar | true
+      - deploy:
+          name: Push Tagged NuCypher Porter Docker Images
+          command: |
+            echo $DOCKER_PASSWORD | docker login -u $DOCKER_USERNAME --password-stdin
+            docker tag nucypher/nucypher-porter:circle nucypher/nucypher-porter:$CIRCLE_TAG
+            docker tag nucypher/nucypher-porter:circle nucypher/nucypher-porter:latest
+            docker push nucypher/nucypher-porter:$CIRCLE_TAG
+            docker push nucypher/nucypher-porter:latest
```
Comment (KPrasch) [inline .circleci/config.yml:995]: Seems a bit redundant, why not `nucypher/porter?`

**Review by KPrasch — APPROVED** [review]
LGTM - Left one comment about naming.

File: .circleci/config.yml (hunk 1 above)
Comment (derekpierre) [reply .circleci/config.yml]: ✔️

File: deploy/docker/porter/docker-compose.yml [hunk 2]
```diff
@@ -3,7 +3,8 @@ version: '3'
 services:
   porter-http:
     restart: on-failure
-    image: nucypher:latest
+    image: porter:latest
```
Comment (vepkenez) [inline deploy/docker/porter/docker-compose.yml:6]: This all looks good

**Review by vepkenez — APPROVED** [review]

**Pull request merged**